### ISSUE TITLE
Accessibility - colors do not have a sufficient contrast ratio

### ISSUE THREAD

**Issue by onemen** [epicweb-dev/kcdshop#55]
Color contrast ratio for body text should be at least 1:4.5 [mdn docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast?utm_source=devtools&utm_medium=a11y-panel-checks-color-contrast#solution)

code blocks and some off the `callout` blocks have much lower contrast ratio.

![color-contrast](https://user-images.githubusercontent.com/3650909/234214533-fecdd42a-51c2-4f1e-b0dc-9f14bf2c0f94.png)

**Comment by onemen** [comment]
[prismjs](https://www.npmjs.com/package/prismjs) has many [examples](https://prismjs.com/download.html)

there is also [highlight.](https://www.npmjs.com/package/highlight.js)
